src/test/java/de/qabel/desktop/DesktopClientGuiTest.java

```diff
@@ -0,0 +1,59 @@
+package de.qabel.desktop;
+
+import de.qabel.core.config.Account;
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class DesktopClientGuiTest {
+
+    private TestKernel kernel;
+    private final AtomicBoolean exited = new AtomicBoolean(false);
+
+    @Test
+    public void notLoggedIn() throws Exception {
```

Comment:
`notLoggedIn` does not describe what you want to test. According to the test code it could be `primaryStageShowsWhenSettingAccountToNull` .. don't know if thats required behavior, but without a better test name I don't know why you are testing this or who might ever set the account to null and why.